esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -1959,13 +1959,11 @@ void EsdlBindingImpl::initEsdlServiceInfo(IEsdlDefService &srvdef)
     //setWsdlAddress(bndcfg->queryProp("@wsdlServiceAddress"));
 
     IProperties *xsdparams = createProperties(false);
-    xsdparams->setProp( "all_annot_Param", 1 );
```

Comment:
Is there a possible code path on which this property (and the one below) is referenced? End users don't need our own extra annotations, but has it been confirmed that we don't?

Reply:
@timothyklemm Yes you were correct. I've pushed up a fix to change the parameter value only when the transaction is an explicit request for the XSD or WSDL. Other uses (such as to generate the web form) will use the default XSL parameters set as above. Please re-review.